nuxt.config.js

```diff
@@ -204,6 +204,7 @@ export default {
     dsn:
       process.env.SENTRY_DSN ||
       'https://3f3e05dbe6994c318d1bf1c8bfcf71a1@o288582.ingest.sentry.io/1525413',
```

Comment:
```suggestion
      'https://53da8fbcebeb48a6bf614a212629df6b@o787041.ingest.sentry.io/5799642',
```

The DSN was for Creative Common's Sentry account 😭. Noting that the Production DSN in our infra is still correct.

Reply:
Was that the reason that it wasn't loading locally? Or is it the reason that we won't see the dev errors in our sentry logs?